packages/terra-menu/src/Menu.jsx

```diff
@@ -89,6 +132,47 @@ class Menu extends React.Component {
     }
   }
 
+  disableContainerChildrenFocus() {
+    const mainDocumentElement = document.querySelector(this.props.rootSelector);
+
+    if (mainDocumentElement) {
+      const inert = +mainDocumentElement.getAttribute('data-overlay-count');
+
+      if (!mainDocumentElement.hasAttribute('data-overlay-count')) {
+        mainDocumentElement.setAttribute('data-overlay-count', '1');
+        mainDocumentElement.setAttribute('inert', '');
+      } else if (mainDocumentElement && mainDocumentElement.hasAttribute('data-overlay-count')) {
+        mainDocumentElement.setAttribute('data-overlay-count', `${inert + 1}`);
+      }
+    }
+  }
+
+  enableContainerChildrenFocus() {
+    const mainDocumentElement = document.querySelector(this.props.rootSelector);
+
+    if (mainDocumentElement) {
+      const inert = +mainDocumentElement.getAttribute('data-overlay-count');
+
+      if (inert === 1) {
+        mainDocumentElement.removeAttribute('data-overlay-count');
+        mainDocumentElement.removeAttribute('inert');
+        if (this.props.triggerElement) {
+          // Shift focus back to element that was last focused prior to opening the menu
+          setTimeout(() => mainDocumentElement.removeAttribute('aria-hidden'), 0);
```

Comment:
Why do we need to use `setTimeout` for this?

Reply:
Did it similar to [abstract modal](https://github.com/cerner/terra-framework/blob/bed6ba818a71d660ae7416724fbd8a3293ecc0c6/packages/terra-abstract-modal/src/AbstractModal.jsx#L150) to make sure aria-hidden is removed.

Reply:
..but why? 🤔Is it that the element might not be available on the DOM?

Reply:
aria-hidden is not removed if setTimeout is not used.